recipes/search_ncbi/meta.yaml

```
@@ -0,0 +1,44 @@
+  sha256: 78bef363451506dd36e29d3cd126840bf67eb6b3eafa6b541076da632c2a43dc  
+
+build:  
+  noarch: python  
+  number: 0  
+  script: "{{ PYTHON }} -m pip install . -vv"  
+  entry_points:  
+    - searchncbi=search_ncbi.cli:main  
+  run_exports:  
+    - {{ pin_subpackage("search_ncbi", max_pin="x.x") }}
+
+requirements:  
```

yamllint

[error] trailing spaces

(trailing-spaces)